<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AcquisitionWindows</class>
 <widget class="QDialog" name="AcquisitionWindows">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1044</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="btn_connection">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Connection</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_LED">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>40</horstretch>
          <verstretch>40</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_startAcqui">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Start Acquisition</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_stopAcqui">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Stop Acquisition</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabwidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_Live">
      <attribute name="title">
       <string>Live</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QWidget" name="widget_3" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QSplitter" name="splitter_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <widget class="QVTKWidget" name="qvtk_Cloud" native="true"/>
             <widget class="QWidget" name="widget_5" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <widget class="QScrollArea" name="scroll_image">
                 <property name="widgetResizable">
                  <bool>true</bool>
                 </property>
                 <widget class="QWidget" name="scrollAreaWidgetContents">
                  <property name="geometry">
                   <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>760</width>
                    <height>318</height>
                   </rect>
                  </property>
                 </widget>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QGroupBox" name="groupBox">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Capture 2D</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QRadioButton" name="rndBtn_captureDepth">
             <property name="text">
              <string>Depth</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rndBtn_captureRGB">
             <property name="text">
              <string>RGB</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>LUT</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QRadioButton" name="rndBtn_oceanLUT">
                <property name="text">
                 <string>Ocean</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="rdnBtn_grayscaleLUT">
                <property name="text">
                 <string>Grayscale</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="rdnBtn_jetLUT">
                <property name="text">
                 <string>Jet</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_SaveCloud">
             <property name="text">
              <string>Save Cloud</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_record">
      <attribute name="title">
       <string>Save</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lbl_infos">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>btn_startAcqui</sender>
   <signal>clicked()</signal>
   <receiver>AcquisitionWindows</receiver>
   <slot>DemarerAcqui()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>796</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>679</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_connection</sender>
   <signal>clicked()</signal>
   <receiver>AcquisitionWindows</receiver>
   <slot>ConnectDevice()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_stopAcqui</sender>
   <signal>clicked()</signal>
   <receiver>AcquisitionWindows</receiver>
   <slot>StopperAcqui()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>934</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>1036</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ConnectDevice()</slot>
  <slot>DemarerAcqui()</slot>
  <slot>StopperAcqui()</slot>
 </slots>
</ui>
